torch/csrc/jit/passes/symbolic_shape_analysis.cpp

```diff
@@ -657,6 +658,14 @@ struct SymbolicShapeGraphAnalyzer {
 
       if (!partial_evaluated_graphs.count(curr)) {
         GRAPH_DEBUG("No graph ", getHeader(curr));
+
+        // If the custom operator does not register shape function,
+        // the custom operator is responsible to handle the dynamic shape
+        // within the func body.
+        if (curr->isMemberOf(tensorexpr::getCustomOperatorSet())) {
```

Comment:
Thanks for the comments. The motivation for this modification is all the custom operator has to register the shape function even it could handle dynamic shapes internally. ConvReLU is a typical case. It supports dynamic shape, but it has to register a dummy shape function to make it could be pulled into the TE group for dynamic shape.

I will remove this code first, do you have any other comments?